content/en/tracing/profiler/search_profiles.md

```diff
@@ -154,12 +154,14 @@ Mutex
 Block
 : Shows the time functions have been waiting on mutexes and channel operations during the profiling period (default: 60s). Sleep, GC, Network and Syscall operations are not captured by this profile. Blocking operations are only captured after they become unblocked, so this profile cannot be used to debug applications that appear to be stuck. For mutex contentions, the stack traces in this profile point to blocked `Lock()` operations. This will tell you where your program is getting blocked, while the mutex profile tells you what part of your program is causing the contention. See our [Block Profiling in Go][1] research for more in-depth information. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles).
 
-Goroutines
+Goroutine
```

Comment:
NIT: I'm okay with renaming this, but we should probably do the same in the UI if we change it in the docs?

![image](https://user-images.githubusercontent.com/15000/151522305-dc6391d5-5877-44dd-90d9-f21c189ccf4b.png)

Reply:
Good call. Many but not all profile types are singular so I was trying to be more consistent there, but let me roll this back for now since I don't have time to update everything.